framework/lib/components/tag/tag.tsx

```diff
@@ -7,18 +7,69 @@ import { TagProps } from './types'
  * @see {@link https://northlight.dev/reference/tag}
  *
  * @example
- * (? <Tag>Brand 1</Tag> ?)
+ * Default size medium.
```

Comment:
Please add a proper documentation with heading ## and and explanation, here and let's make it a bit more professional than just a few words. Remove the "default size is medium"
<img width="968" alt="Screenshot 2023-12-22 at 20 26 25" src="https://github.com/mediatool/northlight/assets/5406237/a32563e1-f846-4262-8ce9-5f14171b5f4d">

Something like

Tag
Tag component is used for items that need to be labeled, categorized, or organized using keywords that describe them.